src/modules/pipelines/mscmextract.bicep

```diff
@@ -0,0 +1,63 @@
+// Description: The extract pipeline for cost management data.
+//              The pipeline is triggered by a storage event.
+//              The extract pipeline queue up the transform pipeline and then exit.
+
+@description('Required. The name of the parent Azure Data Factory..')
+param dataFactoryName string
+
+@description('Required. Name.')
+param pipelineName string
+
+@description('Required. The name of the transform pipeline to execute.')
+param pipelineToExecute string
+
+resource dataFactoryRef 'Microsoft.DataFactory/factories@2018-06-01' existing = {
+  name: dataFactoryName
+}
+
+resource pipeline 'Microsoft.DataFactory/factories/pipelines@2018-06-01' =  {
+  name: pipelineName
+  parent: dataFactoryRef
+  properties: {
+    activities: [
+      {
+        name: 'Execute'
+        type: 'ExecutePipeline'
+        dependsOn: []
+        userProperties: []
+        typeProperties: {
+          pipeline: {
+            referenceName: pipelineToExecute
```

Comment:
Yes.  It's to prevent the extract pipeline failing in ADF if we get more than 100 concurrent files being written to the data lake at the same time.  Concurrent, partitioned exports will do it.  Having the extract pipeline just queue up the transform pipeline works around the issue.

Reply:
Can you add a comment somewhere to explain this somewhere? I don't want someone removing it later because they had the same thoughts I did :-P

Reply:
Done!